com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/ThirdPartyDemand/APS/ApsAndroid.cs

```diff
@@ -46,39 +47,77 @@ public void InitializeNativeSDK() {
 			adRegistration.CallStatic("enableTesting", true);
 		}
 
-		private static Tuple<int, int> AdTypeToDim(AdUnitType type) {
+		private static Tuple<int, int> AdTypeToDim(APSAdUnitType type) {
```

Comment:
If you were to store the width and height of the ad on the `ApsAdSlot` you could remove this function and reference those values directly.